Automattic/wp-calypso
# My Home: Display "Mastering Gutenberg" card

Requires D41978-code.

#### Changes proposed in this Pull Request

- Adds a compact version of the Free Photo Library and Mastering Gutenberg cards to be displayed in the current layout.
- Renders the Mastering Gutenberg card when present in the Home layout API response.
- If both cards are present, displays a dot pager wrapping them.

![Apr-16-2020 16-05-10](https://user-images.githubusercontent.com/1233880/79466138-759e0d80-7ffc-11ea-9a04-99d3263aee24.gif)

#### Testing instructions

- Apply D41978-code and sandbox the API.
- Go to My Home for a site where the selected editor is Gutenberg.
- Make sure you see a dot pager in the right column displaying the Free Photo Library and Mastering Gutenberg cards.
- Open the block editor and switch to the classic editor.
- Go back to My Home.
- Make sure no dot pager is displayed now, and only the Free Photo Library card is present on the right column.

Fixes #41037
